aten/src/ATen/native/cuda/linalg/BatchLinearAlgebra.cpp

```diff
@@ -58,6 +58,8 @@ struct MagmaInitializer {
 } initializer;
 }  // namespace (anonymous)
 
+#define AT_MAGMA_VERSION MAGMA_VERSION_MAJOR*100 + MAGMA_VERSION_MINOR*10 + MAGMA_VERSION_PATCH
```

Comment:
add a check to ensure that magma doesn't ever release a minor/patch 10?

Reply:
Thanks for the suggestion! Added in https://github.com/pytorch/pytorch/pull/77019/commits/42fc6d58ecedf41e6360e4e3a977ba18339b4b6f. Hoping that `#error` is supported in all compilers used in CI.